api/can_api_v2_definition.py

```diff
@@ -191,7 +199,7 @@ class Actuals(base_model.APIBaseModel):
 number of people vaccinated with both the first and second dose.
 """,
     )
-    casesByAge: Optional[Dict[str, int]] = pydantic.Field(None)
+    vaccinesAdministeredDemographics: Optional[Demographics] = pydantic.Field(None)
```

Comment:
Hmm, well here `vaccinesAdministeredDemographics` seems like an okay name, I think because here Demographics is helping to describe a quantity but when it is a class name on its own I'm expecting it to be a noun that describes the stuff in it.

Reply:
Yeah I'll think of a better name for it!